app/models/stocks/marketstack_api.rb

```diff
@@ -0,0 +1,74 @@
+require "net/http"
+require "json"
+
+# Get stock prices from api with:
+# api = Stocks::MarketstackApi.new(api_key: Rails.application.credentials.marketstack_key)
+# api.get_end_of_day_price("KO,SNE,TWX,DIS,SIRI,F,EA,FB,UA,LUV,GPS")
+
+class Stocks::MarketstackApi
+  attr_accessor :api_key
+
+  def initialize(attrs)
+    @api_key = attrs[:api_key] if attrs[:api_key].present?
+  end
+
+  def get_end_of_day_price(symbols, date = "latest")
```

Comment:
likewise, for date, rather than a positional argument

and some kind of documentation on what's valid for a `date: ` param